eclass/meson.eclass

```diff
@@ -314,8 +311,7 @@ meson_src_configure() {
 		: "${BUILD_PKG_CONFIG_PATH:=${PKG_CONFIG_PATH}}"
 	fi
 
-	local mesonargs=(
-		meson setup
+	mesonargs=(
```

Comment:
This is no longer a local variable, so technically speaking it would be more correct to make it uppercase.

Reply:
Sounds reasonable enough.

I'm very much not attached to any particular name here.